openid-4-verifiable-presentations-1_0.md

```diff
@@ -791,6 +791,10 @@ This document also defines the following additional error codes and error descri
 
 - The value of the `request_uri_method` request parameter is neither `get` nor `post` (case-sensitive).
 
+`wallet_unavailable`:
+
+- The Wallet could not respond to the request.
```

Comment:
```suggestion
- The Wallet appears as currently unavailable and therefore unable to process the request.
```

Reply:
The grammar in Guiseppe's suggestion is a little odd; this would be slightly better I think:

```suggestion
- The Wallet appears to be unavailable and therefore unable to respond to the request.
```